### ISSUE TITLE
White-list additions, and benchmark results

### ISSUE THREAD

**Issue by gijs007** [Advantaged/4Kn-Formatting#2]
I was able to convert two of my three NVMe SSD's to 4k native format.
My Crucial T700 and Corsair MP600 Pro NH, which both contain different types of Phison controllers, were successfully converted to 4kn.
On the Crucial T700 this took a little longer, on the Corsair the command was nearly instant.

My Intel Optane 905P only supports 512e mode, and could not be converted. Most consumer/enthusiast Intel SSD's do not support 4kn mode, with the exception of the Intel 750.

I ran some quick benchmarks with Crystal diskmark and AS SSD on Windows 11:
The most significant impact is in latency (measured with AS SSD), although the effect strongly differs between drives:
On the Crucial T700, write latency dropped from 0,034ms to 0,019ms. Read latency dropped form 0,021ms to 0,019ms.
On the Corsair MP600 Pro, write latency was around the same 0,024ms. Read latency dropped from 0,072ms to 0,024ms.

During benchmarking, I also noticed a clear 2-4c drop in max and average temps for these SSD's. 

I got some odd results in the read/write speed test, perhaps 512e mode has more performance fluctuations, it would require more testing to make this clear.
The sequential read performance with Crucial T700 might be slightly better, but this is within the error margin (e.g. 11.900 Mbps vs 12.200 Mbps)
The Corsair MP600 Pro shows better random 4k32t performance (read and write). But here I saw one test in 512e mode with similar results. As such it's unclear if 4kn really improves throughput.

**Comment by Advantaged** [comment]
**Thank you** first for reporting your experiences and _contribute to enhance this project_.
As i know, the Corsair are able to switch to 4Kn, "4 Kibi-Byte-natural", it's mean, the PBS (physical-block-size) is 4-kibi-byte, since end of the "Corsair-series 3". However, with the beginning of Corsair series 4, all NVM-e of this manufacturer can be switched forward & rearward = 4096 and/or 512. How to switch to 512e is not known by myself.

New for me/us, we can add following to the white-list:
* **Crucial T700**
* **Intel 750**

**About bench-mark:**
* The PCI-e bus-speed and/or version (3, 4, 5), play a big role but…
* the biggest "inconvenience: is represented by DT (data-carrier-s) having 512e LBS (logical-block-size).

**Why 512e is worse:**
1. E.g. Samsung NVM-e's: PBS is 512, LBS is 512e, hence the controller lie first to OS telling he's 4096 byte-cluster. Ubuntu, e.g. install the OS on 4096, but than try to convert (on the fly) the own installation to 512 (without "e").
2. 512e LBS-controller must work more and harder (e.g. Samsung), become hot and must reduce the transfer-speed drastically to avoid damage on the hardware after a couple of minutes of "work". i was obligated to make a "low-level-format" on such a controller in order to get PBS=LBS eliminating temperature-, speed- & OS-problems.
3. Last but not least: Each DT-cluster (DT=data-carrier) contains 'data' + 'address-', 'parity-', 'start-', 'stop-' and other bits… storing 4096 bits/byte of data in 512 or 512e LBS data-carrier mean, the DT use a physical-place of 9-10 clusters instead of just 8 (512x8=4096) like a 4k-DT do.

**_By the way:_**
* 512e was specified in year **2012 as bridge/intermediate/provisional** "solution" between 512- & 4096-PBS/LBS data-carrier. Offering today such a technology is retrograde.
* **All modern FS (file-system-s)** like `btrfs`, `zfs`, all servers & also Windows can/must/want install the OS or use a FS with minimum of 4k=4096 bits/bytes.

**In conclusion:**
* DT with a PBS/LBS of 512 and/or 512e are closeouts/remnants/residual-items. Avoid them if possible.
